FFXIVClientStructs/FFXIV/Client/System/Framework/Framework.cs

```diff
@@ -50,6 +64,8 @@ public unsafe partial struct Framework {
 
     [FieldOffset(0x2BF0)] public GameVersion GameVersion;
 
+    [FieldOffset(0x35C0)] public void* SteamApiLibrary;
+
```

Comment:
`SteamApiLibrary` can be omitted, PR #748 is going to add this. 🙂
```suggestion
```

Reply:
Not hte same field, this is the DLL 0x35B8 is the wrapper

Reply:
I see, there is a mistake in the other PR. Sorry.